test/ng/locationSpec.js

```diff
@@ -1601,6 +1601,43 @@ describe('$location', function() {
     });
 
 
+    it('should rewrite links when the specified rewriteLinks attr is detected', function() {
+      configureTestLink({
+        linkHref: 'link?a#b',
+        attrs: 'force-rewrite',
+        html5Mode: {enabled: true, rewriteLinks:false},
+        supportHist: true
```

Comment:
I don't know if you copied this from somewhere else, but `configureTestLink` doesn't seem to support `html5Mode` or `supportHist` :confused: